floaters/src/main/java/com/mdg/droiders/floaters/FloatingViewContainer.java

```diff
@@ -75,4 +80,57 @@ View getCollapsedView() {
     View getExpandedView() {
         return expandedView;
     }
+
+    /**
+     * Init on click listener in the MUSIC PLAYER expanded version of the floating view.
+     *
+     * @param context The @{@link Context} to use for displaying toast on view click
+     */
+    private void initOnClickListeners(final Context context) {
+
+        ImageView playButton = (ImageView) mFloatingView.findViewById(R.id.play_btn);
+        playButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Toast.makeText(context, "Playing the song.", Toast.LENGTH_LONG).show();
+            }
+        });
+
+        ImageView nextButton = (ImageView) mFloatingView.findViewById(R.id.next_btn);
+        nextButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Toast.makeText(context, "Playing next song.", Toast.LENGTH_LONG).show();
+            }
+        });
+
+        ImageView prevButton = (ImageView) mFloatingView.findViewById(R.id.prev_btn);
+        prevButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Toast.makeText(context, "Playing previous song.", Toast.LENGTH_LONG).show();
+            }
+        });
+
+        ImageView closeButton = (ImageView) mFloatingView.findViewById(R.id.close_button);
```

Comment:
\[Checkstyle\] WARNING: 'member def type' have incorrect indentation level 8, expected level should be 4\.